src/lib/modules/server/apn/direct-apns.ts

```
@@ -1,19 +1,14 @@
 import { env } from '$env/dynamic/private';
-import http2, {
-  type ClientHttp2Session,
-  type ClientHttp2Stream,
-  type IncomingHttpHeaders,
-} from 'http2';
-// Direct HTTP/2 APNs implementation with proven JWT library
+import type { ClientHttp2Session, ClientHttp2Stream, IncomingHttpHeaders } from 'http2';
```

ESLint

[error] Expected "http2" (type-import) to come before "$env/dynamic/private" (value-external).

(perfectionist/sort-imports)